compiler/circle-eval-diff/src/FileDataLoader.h

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __CIRCLE_EVAL_DIFF_FILE_DATA_LOADER_H__
+#define __CIRCLE_EVAL_DIFF_FILE_DATA_LOADER_H__
+
+#include "DataSet.h"
+
+#include <dio_hdf5/HDF5Importer.h>
+#include <loco/IR/Node.h>
+#include <luci/IR/CircleNodes.h>
+
+#include <memory>
+#include <string>
+
+namespace circle_eval_diff
+{
+
+using DataType = loco::DataType;
+using Shape = std::vector<loco::Dimension>;
+
+void verifyTypeShape(const luci::CircleInput *input_node, const DataType &dtype,
+                     const Shape &shape);
+uint32_t byteSizeOf(const luci::CircleNode *node);
+
+} // namespace circle_eval_diff
```

Comment:
I guess that you've exposed these methods for testing. If so, `byteSizeOf` is unnecessary to be exposed.

And could you remove `using` keyword in the header file?